repository/git.go

```diff
@@ -116,7 +159,10 @@ func (r *Git) writeConfig() (err error) {
 	if err != nil {
 		return
 	}
-	s := "[credential]\n"
+	s := "[user]\n"
+	s += "name = Konveyor Dev\n"
+	s += "email = konveyor-dev@googlegroups.com\n"
```

Comment:
Please make sure we have a GH Issue to make this configurable on the credentials directly.

Reply:
https://github.com/konveyor/tackle2-addon/issues/21